paasta_tools/cli/cmds/spark_run.py

```diff
@@ -711,6 +726,48 @@ def get_spark_app_name(original_docker_cmd: Union[Any, str, List[str]]) -> str:
     return spark_app_name
 
 
+def _convert_memory_string(value: int, input_unit: str, output_unit: str) -> float:
```

Comment:
this could probably use a more descriptive function name or a docstring - as it is right now, one needs to read the function body to see what this is doing